modules/tensor_mechanics/src/materials/crystal_plasticity/CrystalPlasticityKalidindiUpdate.C

```diff
@@ -24,6 +24,11 @@ CrystalPlasticityKalidindiUpdate::validParams()
   params.addParam<Real>("xm", 0.1, "exponent for slip rate");
   params.addParam<Real>("gss_initial", 60.8, "initial lattice friction strength of the material");
 
+  params.addParam<MaterialPropertyName>(
+      "total_twin_volume_fraction",
+      "Name of the material property of the total twin volume "
+      "fraction, only if twinning is considered in the simulation");
+
```

Comment:
```suggestion
      "total_twin_volume_fraction",
      "Total twin volume fraction, if twinning is considered in the simulation");

```

I'd prefer that over explicitly stating "Name of the material property", which is implied by the type of the parameter, and also a bit misleading, as the user can supply a number value directly.